networkx/algorithms/traversal/tests/test_dfs.py

```diff
@@ -66,3 +66,62 @@ def test_dfs_tree_isolates(self):
         T=nx.dfs_tree(G,source=None)
         assert_equal(sorted(T.nodes()),[1, 2])
         assert_equal(sorted(T.edges()),[])
+
+
+class Testdls:
```

Comment:
This should be `class TestDepthLimitedSearch(object):` --- better name and better for Python 2.7 support.